packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/Datagrid.module.scss

```diff
@@ -16,5 +16,9 @@
   .loading-overlay {
     top: 0;
   }
+
+  .spacer-right-medium {
+    margin-right: 0.7142857143rem;
```

Comment:
Seems like an odd decimal, and doesn't seem to translate into any clean ratio. Why this value?

Reply:
This is dependent on the font-size. If font-size used is 14px, then this would evaluate to 10px.

Reply:
As @benbcai said - rem sizing is a percentage, has nothing to do with how pretty the decimal number is, what is important is how it calculates, so for this case we use 14px as the 1rem base, so in order to achieve 10px, it's 10/14 = 0.7142857143.